Kickstarter-iOS/Views/Controllers/ProjectNavBarViewController.swift

```diff
@@ -108,9 +108,11 @@ public final class ProjectNavBarViewController: UIViewController {
 
     _ = self.shareButton
       |> shareButtonStyle
+      |> UIButton.lens.accessibilityLabel %~ { _ in Strings.dashboard_accessibility_label_share_project() }
 
-    _ = self.starButton
+    _ = self.saveButton
       |> saveButtonStyle
+      |> UIButton.lens.accessibilityLabel %~ { _ in Strings.Save_this_project() }
```

Comment:
So I think that for both the nav bar and the postcard cell, we want to have the `accessibilityLabel` read:
"Toggle saving this project" and then add an `accessibilityValue` that reads "Saved" or "Unsaved". 

Right now this reads "Save this project. Saves this project." which I know is old code, but we may as well fix it now.